docs/running-tests/from-local-system.md

```diff
@@ -7,9 +7,25 @@ The tests are designed to be run from your local computer.
 The test environment requires [Python 2.7+](http://www.python.org/downloads)
 (but not Python 3.x).
 
-On Windows, be sure to add the Python directory (`c:\python2x`, by default) to
-your `%Path%` [Environment Variable](http://www.computerhope.com/issues/ch000549.htm),
-and read the [Windows Notes](#windows-notes) section below.
+<section class="platform">
+
+**Windows**
+
+Generally Windows Subsystem for Linux will provide the smoothest user
+experience for running web-platform-tests on Windows.
+
+Be sure to add the Python directory (`c:\python2x`, by default) to your
```

Comment:
I think it's not obvious that the following section doesn't apply if you are running in WSL. Also with WSL 2, which is more like a built-in VM, I kind of wonder how easy it is to test the Windows binary from WSL. Has anoyone tried this. I *assume* part of the value proposition of WSL is that it should work, but verification seems useful.

Reply:
Hm, I don't know anything about that. I'll see what I can do about setting up a Windows 10 VM locally.